pkg/codegen/main.go

```diff
@@ -106,7 +106,8 @@ func main() {
 				Types: []interface{}{
 					"./pkg/apis/rke.cattle.io/v1",
 				},
-				GenerateTypes: true,
+				GenerateTypes:   true,
+				GenerateClients: true,
```

Comment:
This is actually generating the k8's rke client not the rancher one. So this client would be hitting the Kubernetes api endpoint rather than the rancher. One of the last thing's Jacob had suggested was to use generate the client similar to how the management.cattle.io clients are generated using the Norman code generator. I did it for the provisioning.cattle.io client in this PR.

https://github.com/rancher/rancher/pull/38281